src/common/transformations/src/transformations/op_conversions/convert_topk11_downgrade.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_topk11_downgrade.hpp"
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+#include <openvino/opsets/opset11.hpp>
+#include <openvino/opsets/opset3.hpp>
```

Comment:
```suggestion
#include <openvino/core/rt_info.hpp>
#include <openvino/opsets/opset11.hpp>
#include <openvino/opsets/opset3.hpp>
#include <openvino/pass/pattern/op/wrap_type.hpp>

```

Reply:
I'd rather not touch it since clang format doesn't complain.